src/pudl/output/new_eia923.py

```diff
@@ -0,0 +1,222 @@
+"""Denormalized, aggregated, and filled versions of the EIA 923 tables.
+
+Basic Denormalized EIA-923 assets
+* denorm_generation_eia923 (temporal aggregation)
+* denorm_fuel_receipts_costs_eia923 (temporal aggregation + coalmine)
+* denorm_boiler_fuel_eia923 (temporal aggregation)
+* denorm_generation_fuel_eia923 (temporal aggregation + nuke vs. non-nuke)
+
+More complex denormalized EIA-923 assets:
+* denorm_generation_fuel_all_eia923 (combined nuke and non-nuke assets)
+
+
+Generation Fuel:
+* could also use a nuke vs. non-nuke asset factory
+* Generation Fuel All is derived from nuclear and non-nuclear (separate asset)
+
+Generation:
+* Needs to be able to use allocated net generation from generation fuel table. Should
+  this be the default output?
+
+* Aggregations can be separated from denormalization.
+* Aggregations should only be performed on the normalized tables.
+* Denormalization process is identical on raw, monthly, or annual aggregates.
+* Aggregation asset factory specific to each output table.
+* Hand off aggregated table to denormalization asset / function?
+
+PudlTabl integration considerations:
+* PudlTabl.freq will control what table is pulled from the DB.
+"""
+from typing import Literal
+
+import numpy as np
+import pandas as pd
+from dagster import AssetIn, AssetsDefinition, asset, graph_multi_asset, op
+
+import pudl
+from pudl.metadata.fields import apply_pudl_dtypes
+
+AGG_FREQS = {
+    "AS": "yearly",
+    "MS": "monthly",
+}
+
+
+def generation_eia923_aggregated_asset_factory(
```

Comment:
For the moment I still have separate aggregation factories for each of the output tables. I'm not sure if this is what we want to do.

Some of them need only to run the yearly and monthly (because their base table is created in the main ETL) but some also currently create the base table, because it's novel (so they include `freq=None` too). Does that seem right? Should the unaggregated tables be produced through a different process?